isis/src/qisis/objs/ControlMeasureEditWidget/ControlMeasureEditWidget.cpp

```diff
@@ -769,6 +756,20 @@ namespace Isis {
   }
 
 
+  void ControlMeasureEditWidget::setLeftPosition(double sample, double line) {
```

Comment:
should there be a descriptive header for the setLeftPosition and setRightPosition methods?

Reply:
Added.